Client/Frontend/Fakespot/FakespotOptInCardViewModel.swift

```diff
@@ -23,6 +23,7 @@ struct FakespotOptInCardViewModel {
     let headerTitle: String = .Shopping.OptInCardHeaderTitle
     let headerA11yId: String = AccessibilityIdentifiers.Shopping.OptInCard.headerTitle
     let bodyFirstParagraph: String = .Shopping.OptInCardFirstParagraph
+    let bodyFirstParagraphAmazonOnly: String = .Shopping.OptInCardFirstParagraphAmazonOnly
```

Comment:
In other countries we might support other vendors other than Amazon. To make this more future proof I suggest to rename this:
```suggestion
    let bodyFirstParagraphOneVendor: String = .Shopping.OptInCardFirstParagraphOneVendor
```